src/Microsoft.Azure.SignalR.Management/Configuration/ServiceManagerOptions.cs

```diff
@@ -42,10 +42,10 @@ public class ServiceManagerOptions
         /// </summary>
         public ServiceTransportType ServiceTransportType { get; set; } = ServiceTransportType.Transient;
 
-        // TODO: make obsolete once `ServiceHubContextBuilder.WithNewtonsoftJsonHubProtocol()` are public.
         /// <summary>
         /// Gets the json serializer settings that will be used to serialize content sent to Azure SignalR Service.
         /// </summary>
+        [Obsolete("Use ServiceHubContextBuilder.WithNewtonsoftJsonHubProtocol() or ServiceManagerBuilder.WithNewtonsoftJsonHubProtocol() instead. This property only affects transient mode.")]
```

Comment:
what does `This property only affects transient mode.` mean? give advice for persistent mode?